pyradiosky/utils.py

```diff
@@ -278,3 +280,113 @@ def download_gleam(path=".", filename="gleam.vot", overwrite=False, row_limit=No
     table.write(opath, format="votable", overwrite=overwrite)
 
     print("GLEAM catalog downloaded and saved to " + opath)
+
+
+def modified_gleam(
+    gleam_filename="gleamegc.dat",
```

Comment:
Where do you get this file from? We have a method in this utils module to download the GLEAM catalog as a VOT table from Vizier. But it isn't just a text file like it looks like you're expecting this one to be.